static/app/views/explore/tables/aggregatesTable.tsx

```diff
@@ -118,12 +118,14 @@ export function AggregatesTable({}: AggregatesTableProps) {
             result.data?.map((row, i) => (
               <TableRow key={i}>
                 {fields.map((field, j) => {
+                  const column = eventView.getColumns()[j];
```

Comment:
Why do you need an `EventView` here? We're trying to keep this isolated as much as possible as exposing it in too many places has caused issues before.

Reply:
All of the logic required to form the `TableColumn<keyof TableDataRow>` type that's passed down to cellAction (for determining allowed actions) belongs in `EventView` right [here](https://github.com/getsentry/sentry/blob/master/static/app/views/discover/utils.tsx#L86-L124).